liquidprompt

```diff
@@ -1289,6 +1291,28 @@ _lp_dirstack_color() {
     lp_dirstack_color="${LP_COLOR_DIRSTACK}${LP_MARK_DIRSTACK}${lp_dirstack}${NO_COL}"
 }
 
+_lp_aws_profile() {
+    (( LP_ENABLE_AWS_PROFILE )) || return 2
+
+    local ret
+
+    if [[ -n "${AWS_PROFILE-}" ]]; then
+        __lp_escape "${AWS_PROFILE##*/}"
+        lp_aws_profile=$ret
+    elif [[ -n "${AWS_DEFAULT_PROFILE-}" ]]; then
+        __lp_escape "${AWS_PROFILE##*/}"
+        lp_aws_profile=$ret
+    else
+        return 1
+    fi
+}
+
+_lp_aws_profile_color() {
+    _lp_aws_profile || return "$?"
+
+    lp_aws_profile_color="[${LP_COLOR_AWS_PROFILE}AWS:${lp_aws_profile}${NO_COL}]"
```

Comment:
This `AWS:` string really should be turned into a "mark" configuration option, probably named `LP_MARK_AWS_PROFILE`.

Reply:
You are right, it should but I have decided to remove this altogether; I added it as an afterthought and returning to it I think it is unnecessary and clutters the prompt.

If there was a desire for a marker, one could be added in a separate commit at a later stage.